applications/zkSBT.md

```diff
@@ -0,0 +1,160 @@
+
+For AsMatch Users:
+
+* Install polkadot.js wallet and Manta wallet, and create key pairs for both of these two wallets
+* The user uploads the photo as an SBT picture and the AsMatch back-end service will generate similar AIGC pictures with different styles
+* The user selects one of the AIGC pictures and signs the mint transaction to the Polkadot blockchain
+* The user sign transaction by Manta wallet to sync SBT metadata from Polkadot blockchain to Manta
+* The user generates a proof key of SBT on the AsMatch PC website
+* The user opens AsMatch APP on iOS/Android, and copy-paste or scans the QR code of the proof key to verify the ownership of SBT
+* The user set SBT picture as the profile photo
+
+PC website:
+
+![](https://raw.githubusercontent.com/As-Match/documentation/main/images/UploadPhotos.png)
+![](https://raw.githubusercontent.com/As-Match/documentation/main/images/SelectTheme.jpg)
+![](https://raw.githubusercontent.com/As-Match/documentation/main/images/Minted.png)
+
+iOS/Android APP:
+
+![](https://raw.githubusercontent.com/As-Match/documentation/main/images/EnterProofKey.jpg)
+![](https://raw.githubusercontent.com/As-Match/documentation/main/images/BindzkSBT.png)
+![](https://raw.githubusercontent.com/As-Match/documentation/main/images/BindzkSBT2.png)
+
+### Ecosystem Fit
+
+The SBT is playing more and more important roles in the Web3 World. SBTs have a wide range of potential use cases. Here are some examples that could potentially find use in everyday life.
+
+* KYC proof - For example, [Binance Account Bound (BAB)](https://www.binance.com/en/support/faq/how-to-mint-binance-account-bound-bab-token-bacaf9595b52440ea2b023195ba4a09c) tokens are the first-ever Soulbound Token (SBT) built on the BNB Chain. BAB token is non-transferable and has no monetary value. It will function as a digital verification tool for Binance users who have completed Identity Verification.
+
+* Education history - When people graduate from university, they receive a certificate proving completion of the required courses. The university could be a Soul issuing the SBTs, and the students would be Souls on the receiving end. The SBT would store the student’s credentials, proving they hold the relevant qualifications and are a member of the university. Simply put, the SBT would function as proof of attendance.
+
+* Job applications – In theory, job applicants could submit all their prior work history and professional certificates using official SBTs issued by previous companies and institutions. The SBTs would function as proof of skill certificates.
+
+After all, the zkSBT that we are proposing is not only a use case for AsMatch but also a best SBT practice for the Polkadot ecosystem. The ink SBT smart contract we are developing is a good example for Dapps which also want to deploy SBT on the Polkadot blockchain. And we are ready to integrate more SBTs from the Polkadot blockchain into AsMatch users.
```

Comment:
Just to reiterate, ink! contracts can't be deployed to Polkadot, and would have to be deployed to a parachain. But I'm not sure if Manta supports ink! or not yet at this point.